src/chi/components/steps/steps.scss

```diff
@@ -0,0 +1,84 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .a-steps {
+    display: table;
+    padding: 0;
+    table-layout: fixed;
+    width: 100%;
+
+    & > li {
+      color: $brand-color;
+      display: table-cell;
+      font-size: $text;
+      position: relative;
+      text-align: center;
+      z-index: 0;
+
+      &:before {
+        background-color: $text-white;
+        border-radius: 50%;
+        content: '\25CF';
+        display: block;
+        font-size: 1.85rem;
+        height: 1.125rem;
+        line-height: 1rem;
+        margin: 0 auto 0.25rem;
+        width: 1.125rem;
+      }
+
+      &:after {
+        background-color: $brand-color;
+        content: '';
+        height: 0.125rem;
+        left: 50%;
+        position: absolute;
+        top: 0.5rem;
+        width: 100%;
+        z-index: -1;
+      }
+
+      &:last-child {
+        &:after {
+          display: none;
+        }
+      }
+
+      &.-active {
+        & a {
+          color: $brand-color;
+        }
+
+        &:before {
+          box-shadow: 0 0 0 0.125rem $brand-color;
+        }
+
+        &:after {
+          background-color: set-color(grey, 40);
+        }
+
+        & ~ li {
+          color: set-color(grey, 40);
+
+          & a {
+            color: $muted-color;
+          }
+
+          &:before {
+            border-color: $text-white;
+          }
+
+          &:after {
+            background-color: set-color(grey, 40);
+          }
+        }
+      }
+    }
+
+    & a {
+      color: $text-color;
```

Comment:
@davidojedacl remove text-decoration when hover or change the html tag to span or p

Reply:
anchor tag is fine, let's just set `text-decoration: none;`

Reply:
@davidojedacl actually, can we set all links in the steps component to `text-decoration: none;` ?

Reply:
ok